reference/forms/types/timezone.rst

```diff
@@ -105,6 +129,8 @@ Inherited Options
 
 These options inherit from the :doc:`ChoiceType </reference/forms/types/choice>`:
 
+.. include:: /reference/forms/types/options/choice_translation_domain.rst.inc
```

Comment:
I think we should add a note that this option can only be used when `intl` is set to `true`.

Reply:
I thought about that ... however, I saw that the code shows a perfect error message when you do that (see https://github.com/symfony/symfony/pull/31294/files#diff-c6198934d4f35bc102ff362d40b71285R72) so I guess it's fine to not add these implementation details in the docs.

Reply:
It's also more or less explained above. Works for me :)

Reply:
My thought is that people will just notice this option and try to use it. But it's not a big deal. We can add something later anyway if we receive complaints about it not being straight-forward (the error provided by the form type is quite clear indeed).